calico_containers/calico_ctl/container.py

```diff
@@ -7,13 +7,20 @@
   calicoctl container remove <CONTAINER>
 
 Description:
-  Add or remove containers to calico networking, manage their IP addresses and profiles.
+  Add or remove containers to Calico networking, manage their IP addresses and profiles.
   All these commands must be run on the host that contains the container.
 
 Options:
+  <IP>                     The IP address desired. If "ipv4", "ipv6", or a CIDR
+                           is given, then Calico will attempt to automatically
+                           assign the first available IPv4 address, IPv6
```

Comment:
"the first available" -> "an available"

I don't want to imply anything concrete about allocation strategy.  It currently obeys host affinity, so will not, in general, pick the lowest address in a pool.  We may later implement other changes to allocation strategy.